tasks/nfs-clients.yml

```diff
@@ -11,5 +11,6 @@
   mount:
     path: "{{ nfs_client_mnt_point}}"
     src: "{{ nfs_server }}:{{ nfs_export }}"
+    opts: "{{ nfs_client_mnt_options }}"
```

Comment:
Does this work with the default value or do we need `opts: "{{ nfs_client_mnt_options | default(omit, true)}}"`?

Reply:
I have not tested. It did work previously without this variable with defaults. We want to pass options since by default CentOS mounts as nfsv4 given chance on the server side